src/utils/ui.js

```diff
@@ -0,0 +1,241 @@
+import { getState, saveState } from './state';
+import { makeElement } from './misc';
+import * as player from './player';
+
+function createBlockList() {
```

Comment:
Trying not to have to reread every single line of code in these files since a lot of it was just moved around.

Is there anything in this file that you notably changed? If so, could you leave a brief comment in the PR itself denoting that? Will really help with reviewing,thanks man!

Reply:
Not a problem! Mainly just copying and pasting functions over, but I've added comments to `src/util/ui.js` for all changes